bodhi/server/migrations/versions/e383591ed939_add_package_manager_and_testing_.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2019 Mattia Verga
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+Add package_manager and testing_repository columns to the Release model.
+
+Revision ID: e383591ed939
+Revises: c98beb4940b5
+Create Date: 2019-03-22 16:48:47.916552
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = 'e383591ed939'
+down_revision = 'c98beb4940b5'
+
+
+def upgrade():
+    """Add package_manager Enum type and testing_repository string type.
+
+    Also set the right values for items already in database.
+    """
+    package_manager = postgresql.ENUM('dnf', 'yum', name='ck_package_manager')
+    package_manager.create(op.get_bind())
+    op.add_column('releases',
+                  sa.Column('package_manager',
+                            postgresql.ENUM('dnf', 'yum', name='ck_package_manager'),
+                            nullable=False,
```

Comment:
There are releases in Bodhi that do not use dnf or yum, such as flatpaks and containers. We might want to allow this to be nullable since we don't have support for update instructions on those releases yet. We could allow null to mean that the release is not supported.

Or another idea is to call this a "yum_package_manager" enum, and make a YumRelease model and have that model be the only one that has this field. That would be a lot more work though…

Reply:
I have added an 'unspecified' value as default. I think is more logical to avoid having an Enum default to Null, let me know if it's a problem.